packages/web/config/wallet-registry.ts

```diff
@@ -285,4 +285,73 @@ export const CosmosWalletRegistry: CosmosRegistryWallet[] = [
     },
     features: [],
   },
+  {
+    ...CosmosKitWalletList["trust-extension"],
+    logo: "/wallets/trust.png",
+    lazyInstall: () =>
+      import("@cosmos-kit/trust-extension").then((m) => m.TrustExtensionWallet),
+    windowPropertyName: "trustwallet",
+    async supportsChain(chainId) {
+      const trustAvailableChains: MainnetChainIds[] = [
+        "cosmoshub-4",
+        "osmosis-1",
+        "stride-1",
+        "neutron-1",
+        "axelar-dojo-1",
+        "laozi-mainnet",
+        "columbus-5",
+        "phoenix-1",
+        "evmos_9001-2",
+        "injective-1",
+        "stargaze-1",
+        "crypto-org-chain-mainnet-1",
+        "kava_2222-10",
+      ];
+      return trustAvailableChains.includes(chainId as MainnetChainIds);
+    },
+    stakeUrl: "https://trustwallet.com/staking",
+    governanceUrl: "https://governance.trustwallet.com/",
+    features: [],
+    mode: "extension", // Add mode property with correct value
+  },
+  /* {
+    ...CosmosKitWalletList["trust-mobile"],
+    logo: "/wallets/trust.png",
+    mobileDisabled: false,
+    lazyInstall: () =>
+      import("~/integrations/trust-walletconnect").then(
+        (m) => m.TrustMobileWallet
+      ),
+    supportsChain: async (chainId) => {
+      const trustMobileAvailableChains: MainnetChainIds[] = [
+        "akashnet-2",
+        "mantle-1",
+        "axelar-dojo-1",
+        "cosmoshub-4",
+        "emoney-3",
+        "evmos_9001-2",
+        "injective-1",
+        "irishub-1",
+        "juno-1",
+        "kava_2222-10",
+        "likecoin-mainnet-2",
+        "mars-1",
+        "neutron-1",
+        "osmosis-1",
+        "secret-4",
+        "pacific-1",
+        "sentinelhub-2",
+        "stargaze-1",
+        "iov-mainnet-ibc",
+        "stride-1",
+        "phoenix-1",
+      ];
+
+      return trustMobileAvailableChains.includes(chainId as MainnetChainIds);
+    },
+    stakeUrl: "https://trustwallet.com/staking",
+    governanceUrl: "https://governance.trustwallet.com/",
+    features: [],
+    mode: "wallet-connect",
+  },*/
```

Comment:
Do we need to keep this comment in here?

Reply:
@jonator 
Although I don't really need it, I didn't delete it because I thought we would use it in the future
If you want, I will delete it

Reply:
I think we should delete it. We have version control to get it back if need be. After this it looks good to go

Reply:
@MaxMillington I removed that part
Would you check it? cc: @jonator